### ISSUE TITLE
Prevent creation of default alert rule

### ISSUE THREAD

**Issue by landorg** [jianyuan/terraform-provider-sentry#329]
Hi.
Is there a way to prevent sentry from creating the default alert rule for a new project?

Currently I create an extra rule after project creation but the default rule can't be disabled or deleted and we have to manually delete it.

Thought about using the new import tags but I'd need to get the rule id somehow for the import:
```
terraform import sentry_metric_alert.default org-slug/project-slug/rule-id
```

Thanks

**Comment by a-suenami** [comment]
I encounter the same problems and also hope to avoid creating the default alert setting. 😅

**Comment by slobo** [comment]
["Create New Project" endpoint](https://docs.sentry.io/api/projects/create-a-new-project/) takes a `default_rules` parameter. It would be nice if we could either supply it in the `sentry_project` resource, or if it defaulted to `false` so that default rules are never created

**Comment by jianyuan** [comment]
The `default_rules` parameter is now available in [v0.12.0](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project#default_rules)!
